python/source/robot_model/bind_model.cpp

```diff
@@ -30,7 +30,14 @@ void model(py::module_& m) {
 
   py::class_<Model> c(m, "Model");
 
-  c.def(py::init<const std::string&, const std::string&>(), "Constructor with robot name and path to URDF file.", "robot_name"_a, "urdf_path"_a);
+  c.def(py::init([](const std::string& robot_name, const std::string& urdf_path, const std::vector<std::string>& geometry_package_paths) {
```

Comment:
Btw why is this now a lambda function? Did you have to do that because of the default value for the last argument?

Reply:
Yes exactly, this is due the last argument that is default